File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.py
import sys

num = int(input())
list= [int(sys.stdin.readline()) for i in range(num)]

# for i in range(num):
#     input_num = int(input())
#     list.append(input_num)

list.sort()

for i in range(num):
    print(list[i])

File: 백준/Silver/2751. 수 정렬하기 2/수 정렬하기 2.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"sort"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	writer := bufio.NewWriter(os.Stdout)
	defer writer.Flush()

	var size int
	fmt.Fscan(reader, &size)
	nums := make([]int, size)

	for i := 0; i < size; i++ {
		fmt.Fscan(reader, &nums[i])
	}

	sort.Ints(nums)

	for _, num := range nums {
		fmt.Fprintln(writer, num)
	}
}
